mod/gc_streaming_content/views/default/css/css.php

```diff
@@ -15,15 +15,15 @@
 
          
      }  
-     .stream-new-wire:hover{
+     .stream-new-wire:hover, .stream-new-newsfeed:hover{
          text-decoration: underline;
          cursor: pointer;
      }
      
-     .posts-holder ul:first-child{
+     .posts-holder, .newsfeed-posts-holder ul:first-child{
          border-left: 2px solid #9bc1d4;
      }
      
-     .posts-holder ul .list-break{
+     .posts-holder, .newsfeed-posts-holder ul .list-break{
```

Comment:
This is adding an extra border bottom on the wire.
Try:
.post-holder ul .list-break, .newsfeed-posts-holder ul .list-break